### PR TITLE
[MWPW-135726] Embed Vimeo and YouTube Videos in Column Block

### PR DESCRIPTION
**Pull request by toka1300**
Video links are converted to social link thumbnails by default. I have an added an exception to this, which is when the link is authored with #embed-video in the link.

Also, updated all files to import the buildCarousel function correctly.

Resolves: [MWPW-135726](https://jira.corp.adobe.com/browse/MWPW-135726)

Test URLs:
- Before: https://stage--express--adobecom.hlx.page/drafts/casey/embed-video/embedded-inline-vimeo
- After: https://dotcom-99657--express--adobecom.hlx.page/drafts/casey/embed-video/embedded-inline-vimeo?lighthouse=on

### CODE REVIEW

File: express/blocks/columns/columns.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ import {
   addHeaderSizing,
 } from '../../scripts/scripts.js';
 import { addFreePlanWidget } from '../../scripts/utils/free-plan.js';
+import { embedYoutube, embedVimeo } from '../embed/embed.js';
```
Comment (qiyundai) [inline express/blocks/columns/columns.js:23]: Is this import from the right place?

**Review by qiyundai — CHANGES_REQUESTED** [review]
I think we are missing the commit for refactoring embed-videos.js?

**Comment by vhargrave** [comment]
Wow, what a jump in performance. Nice work! 👏 
Do we need something for youtube videos too or just for vimeo videos?

**Comment by qiyundai** [comment]
> Wow, what a jump in performance. Nice work! 👏 Do we need something for youtube videos too or just for vimeo videos?

I think we can treat that as a fast follow. Or, if you think your POC branch code is ready for review we can just raise a PR from there! 💪

**Comment by vhargrave** [comment]
> > Wow, what a jump in performance. Nice work! 👏 Do we need something for youtube videos too or just for vimeo videos?
> 
> I think we can treat that as a fast follow. Or, if you think your POC branch code is ready for review we can just raise a PR from there! 💪

Ok, I see the confusion. I never intended on merging my POC branch. It's sole purpose was just to quickly show a better way to add these iframes. Happy to have it worked on in a separate task though, but let's have this separate story created and linked here so that it's not forgotten.

**Review by apganapa — APPROVED** [review]

**Review by JingleH — APPROVED** [review]

**Pull request merged**